package.json

```diff
@@ -26,7 +26,7 @@
     "browserify-min": "browserify index.js --standalone faunadb | terser -c -m --keep-fnames --keep-classnames -o dist/faunadb-min.js",
     "prettify": "prettier --write \"{src,test}/**/*.{js,ts}\"",
     "test": "jest --env=node --verbose=false ./test",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release --tag-format ${version}"
```

Comment:
@fauna-rodrigo See these docs: https://semantic-release.gitbook.io/semantic-release/usage/configuration#tagformat

I was originally trying to add the flag in the `"release"` field like our other options, however, the docs don't show how to reference the tag form properly in the JSON 🤷